bolt-modules/out/spec/functions/out/verbose_spec.rb

```diff
@@ -0,0 +1,201 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+require 'bolt/executor'
+require 'bolt/inventory'
+
+describe 'out::verbose' do
+  let(:executor)  { Bolt::Executor.new }
```

Comment:
You can also copy the edits to the `message` spec test in Tom's PR so these variables aren't needed. Since the function is so simple it should just test:
- That `executor.publish_event` is called with the right parameters
- That analytics are sent
- That the function can't be called when `Puppet[:tasks]` isn't set